.github/workflows/ci.yml

```
@@ -23,7 +23,7 @@ jobs:
     - name: Install .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 3.0.100
+        dotnet-version: 3.1.403
```

```suggestion
        dotnet-version: '3.1.x'
```

Use the above instead to be more protected against .NET versions